client/src/components/PacmanGame/PacmanGame.jsx

```diff
@@ -131,7 +136,9 @@ class PacmanGame extends Component {
   }
 
   startGame = () => {
-    const { config, pacman, status } = this.state;
+    const {
+      config, pacman, status, playerId,
+    } = this.state;
     if (status === 1) {
```

Comment:
Create a status file for statuses.

Reply:
It is already there on the client. Will do it when we will move this to the server.